platforms/shared/configuration/roles/setup/certmanager/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+---
+##Checks if cert-manager is installed 
```

Comment:
This can be renamed to `# Checks if cert-manager is installed` (single #, space between # and task description)

Reply:
fixed in pr #2187 as unwanted files got added while squashing